src/test/java/org/opensearch/knn/index/KNNCircuitBreakerIT.java

```diff
@@ -48,7 +48,7 @@ private void tripCb() throws Exception {
         createKnnIndex(indexName2, settings, createKnnIndexMapping(FIELD_NAME, 2));
 
         Float[] vector = { 1.3f, 2.2f };
-        int docsInIndex = 5; // through testing, 7 is minimum number of docs to trip circuit breaker at 1kb
+        int docsInIndex = 7; // through testing, 7 is minimum number of docs to trip circuit breaker at 1kb
```

Comment:
It's weird though, setting this to 5, it was slightly less than 1KB making this test fail.